control/server.py

```diff
@@ -69,6 +69,10 @@ def __init__(self, config):
         if not self.name:
             self.name = socket.gethostname()
         self.logger.info(f"Starting gateway {self.name}")
+        # Configure linux core-dump file size
+        rc = os.system("ulimit -c unlimited")
```

Comment:
@leonidc , setting limit in python code could be removed, core limit unlimited is set in docker-compose.yml, see nvmeof-base change